books/free-programming-books-th.md

```diff
@@ -32,9 +32,13 @@
 
 * [โครงสร้างข้อมูลฉบับวาจาจาวา](https://www.cp.eng.chula.ac.th/books/ds-vjjv/) - สมชาย ประสิทธิ์จูตระกูล
 * [แบบฝึกปฏิบัติฉบับวาจาจาวา](https://www.cp.eng.chula.ac.th/books/java101-lb/) - สมชาย ประสิทธิ์จูตระกูล
+* [เริ่มเรียนเขียนโปรแกรม: ฉบับวาจาจาวา](https://www.cp.eng.chula.ac.th/books/bp-vjjv/) - สมชาย ประสิทธิ์จูตระกูล
 * [Java Programming Concept](http://it.e-tech.ac.th/poohdevil/JavaConcepts/) - Rungrote Phonkam
 
-
 ### Python
 
 * [Python ๑๐๑](https://www.cp.eng.chula.ac.th/books/python101/) - กิตติภณ พละการ, กิตติภพ พละการ, สมชาย ประสิทธิ์จูตระกูล, สุกรี สินธุภิญโญ
+* [เชี่ยวชาญการเขียนโปรแกรมด้วยไพธอน Programming expert with Python](https://www.theconfig.me/p/programming-expert-with-python.html) - ผศ.สุชาติ คุ้มมะณี
```

Comment:
Can't tell if this is an authorized distribution. The website does not appear to be relate to the author

Reply:
OK